examples/fabric/nested_ingestion_pipeline/fabric/tests/test_pl_ingestion_activity.py

```diff
@@ -0,0 +1,89 @@
+import os
+from pathlib import Path
+
+import pytest
+from data_factory_testing_framework import TestFramework, TestFrameworkType
+from data_factory_testing_framework.models import Pipeline
+from data_factory_testing_framework.state import (
+    PipelineRunState,
+    RunParameter,
+    RunParameterType,
+)
+
+
+@pytest.fixture
+def test_framework(request: pytest.FixtureRequest) -> TestFramework:
+    return TestFramework(
+        framework_type=TestFrameworkType.Fabric,
+        # root_folder_path = os.path.join(Path(__file__).parent,"pl_ingestion.DataPipeline")
+        root_folder_path=os.path.join(Path(request.fspath.dirname).parent, "pl_ingestion.DataPipeline"),
```

Comment:
Given that we recommend to set the root_folder_path to the root (!) of the data_factory repository, I would suggest to change the root_folder_path to: 

`os.path.join(Path(request.fspath.dirname).parent)`

Same for some other test files, but I will not comment there.

Reply:
nit: Maybe also good to remove the commented out code

Reply:
Good catch! I forgot to change it at the end. I was simplifying the path while testing so it didn't ran on all folders. Done!